gclient/src/api/mod.rs

```diff
@@ -61,28 +66,38 @@ impl GearApi {
         ))
     }
 
+    /// Create and init a new `GearApi` instance that will be used with the
+    /// local node working in developer mode (running with `--dev` argument).
     pub async fn dev() -> Result<Self> {
         Self::init(WSAddress::dev()).await
     }
 
+    /// Create and init a new `GearApi` instance that will be used with the
+    /// public Gear node.
     pub async fn gear() -> Result<Self> {
         Self::init(WSAddress::gear()).await
     }
 
+    /// Create and init a new `GearApi` instance that will be used with the
+    /// public Vara node.
     pub async fn vara() -> Result<Self> {
         Self::init(WSAddress::vara()).await
     }
 
-    // This stuff to be considered.
+    /// Create an [`EventListener`] to process incoming events.
     pub async fn subscribe(&self) -> Result<EventListener> {
         let events = self.0.finalized_blocks().await?;
         Ok(EventListener(events))
     }
 
+    /// Set the number used once (`nonce`) that will be used in extrinsics.
```

Comment:
while sending*
I'd also add some comments how nonce works and interact with tx pool

Reply:
Done